src/App.tsx

```diff
@@ -1,61 +1,113 @@
+import { FormEventHandler, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Task } from './react-app-env';
+
+function getNextTaskId(tasks:Task[]) {
+  return tasks.sort((task1, task2) => task2.id - task1.id)[0].id + 1;
```

Comment:
Don't name variables like `task1` and `task2` because it's not very understandable and doesn't look good, lets go with `firstTask` and `secondTask`